_sass/components/_footer.scss

```diff
@@ -1,161 +1,236 @@
+    @include at-media("desktop") {
+      min-height: 17rem;
     }
-  }
 
-  .tts {
-    padding-right: 2rem;
-  }
+    a {
+      text-decoration: none;
+      display: block;
+      margin: 0 0 1em 0;
+      white-space: nowrap;
+    }
 
-  .questions_addl_info {
-    padding: 1em;
-    background: $blue;
-    min-width: 23rem;
+    .navigation {
+      background: url(../img/logo_sign_in.svg) top left no-repeat;
+      display: block;
+      justify-content: flex-end;
+      padding-top: ($container-spacing-sm * 3);
 
-    ul {
-      margin: 0;
-      padding-left: 1em;
+      @include at-media("tablet") {
+        display: flex;
+        padding-top: 0;
+      }
     }
-  }
-}
 
-.footer-usa-gov {
-  padding: 1em 0;
-  background-color: darken($navy, 7%);
-  color: $white;
+    .grid {
+      display: flex;
+      justify-content: space-between;
+      border-bottom: 1px solid $border-color;
+      flex-wrap: wrap;
+    }
 
-  .contact-usa-gov {
-    display: inline-block;
-    margin-left: 1rem;
-    text-decoration: none;
-    padding: 0.75rem;
-    width: unset;
-  }
-}
+    .grid section {
+      margin-right: 1rem;
 
-.learn-more {
-  background-color: transparent;
-  color: $white;
-  box-shadow: inset 0 0 0 1px $white;
-  white-space: nowrap;
-  flex: 0;
-  width: auto;
-  margin: 1em 0;
-  padding: 0.75rem;
-  font-weight: normal;
-
-  &:hover {
-    background-color: transparent;
-    color: $white;
-    box-shadow: inset 0 0 0 1px $white;
-  }
+      @include at-media("tablet") {
+        margin-right: 3rem;
 
-  &::after {
-    content: '';
-    display: inline-block;
-    width: 0.7rem;
-    height: 0.7rem;
-    background: url(../img/angle-arrow-down-white.svg) center/contain no-repeat;
-    margin-top: 0;
-    margin-left: 0.5rem;
-  }
+        &:last-child {
+          margin-right: 0;
+        }
+      }
+    }
 
-  &[aria-expanded=true] {
-    &::after {
-      background: url(../img/angle-arrow-up-white.svg) center/contain no-repeat;
+    .grid section h6 {
+      margin: 0 0 1em 0;
+      font-size: $base-font-sm;
+      color: $blue-mid;
     }
-  }
-}
 
-.product-mgmt-statement {
-  max-width: 37rem;
-  margin: 1em 0;
-  color: $white;
-  display: flex;
+    .grid section a {
+      color: $blue-mid;
+      font-size: $base-font-sm;
+    }
 
-  svg {
-    flex: none;
-    margin-right: 1em;
-  }
+    .system-status {
+      display: flex;
+      align-items: center;
+      padding-top: $container-spacing-sm;
+
+      &::before {
+        content: "";
+        width: 1.75rem;
+        height: 1.75rem;
+        display: inline-block;
+        margin-right: 0.25rem;
+        color: $blue;
+        background: url(../img/system-status/icon-system-status.svg) left center
+          no-repeat;
+      }
 
-  p {
-    margin: 0;
-    padding: 0;
+      a {
+        margin: 0;
+        font-size: $base-font-sm;
+        color: $blue-mid;
+      }
+    }
   }
-}
 
-@media #{$breakpoint-menu} {
   .footer-gsa {
```

Comment:
Something with these style revisions appear to cause the GSA portion of the footer to no longer align with the grid boundary.

Before|After
---|---
![before](https://user-images.githubusercontent.com/1779930/99005043-11014d80-250e-11eb-9e84-c5f815d74a3f.png)|![after](https://user-images.githubusercontent.com/1779930/99005049-15c60180-250e-11eb-91cb-4299ba0349a9.png)